### PR TITLE
game of thrones reference removal.

### PR DESCRIPTION
**Pull request by russkidooski**

### CODE REVIEW

**Comment by AydinAdn** [comment]
I hope it gets merged... makes me cringe every time I read it

**Comment by MS1LVA** [comment]
I'm on the fence about this.

File: whitepaper/README.md [hunk 1]
```diff
@@ -8,9 +8,6 @@ Bruce Fenton<br>
 Tron Black<br>
 http://www.ravencoin.org<br>
 3rd April 2018
-</p>
-<p align="center"><i>
-In the fictional world of Westeros, ravens are used as messengers who carry statements of truth. Ravencoin is a use-case focused blockchain designed to carry statements of truth about who owns what assets.
 </i></p>
```
Comment (zachgibb) [inline whitepaper/README.md]: this closing `</i>` would need to be removed as well

**Comment by bedri** [comment]
NO, NO and NO!!! Instead of removing this reference, this MUST be emphasized more and more! I totally disagree with this pull request. The dreams are not there to be murdered but for to keep ALIVE!

File: whitepaper/README.md (hunk 1 above)
Comment (russkidooski) [reply whitepaper/README.md]: thanks for pointing that out

Comment (zachgibb) [reply whitepaper/README.md]: no problem!

**Comment by ShortsellerEnrichmentCommission** [comment]
I prefer to have it removed. It's cringe worthy.

**Comment by russkidooski** [comment]
can we get any update on this? in my opinion it makes us look unprofessional.

**Comment by TronBlack** [comment]
I'm not weighing in on this.  But I'll gather some more opinions so we can get this resolved (closed or accepted).

**Comment by BruceFenton** [comment]
Thank you-  a couple ideas.

We can easily remove it in the future but changing a paper that's already published is not the best precedent

The paper doesn't need to be in GitHub actually as it just serves as a reference for ideas at the time it was published.

It becomes less relevant as time goes on and the community and use cases evolve.

**Comment by BruceFenton** [comment]
One idea is to just make an alternative version without that language.

Better idea is we change the name and make it more of an ongoing / open document and we can easily change it

(the old paper could be more of a historical reference)

**Pull request closed without merging**